PatientWebApplication/UserMicroserviceApi/Model/DoctorUser.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+
+namespace UserMicroserviceApi.Model
+{
+    public class DoctorUser : EmployeeUser
+    {
+        public bool isSpecialist { get; set; }
+
+        public virtual List<DoctorNotification> specialNotifications { get; set; }
+        public string ordination { get; set; }
+
+        public string speciality { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).